core/templates/dev/head/pages/collection_editor/collection_editor.html

```diff
@@ -129,6 +129,7 @@
 
   <script src="{{TEMPLATE_DIR_PREFIX}}/services/ExplorationHtmlFormatterService.js"></script>
   <script src="{{TEMPLATE_DIR_PREFIX}}/services/LocalStorageService.js"></script>
+  <script src="{{TEMPLATE_DIR_PREFIX}}/services/ExtensionTagAssemblerService.js"></script>
```

Comment:
Move this above the previous import, we want to keep the imports in alphabetical order.